src/attributes.cpp

```diff
@@ -2196,7 +2197,7 @@ namespace attributes {
     }
 
     std::string CppPackageIncludeGenerator::getHeaderGuard() const {
-        return "__" + packageCpp() + "_h__";
+        return "RCPP_" + packageCpp() + "_h_gen_";
```

Comment:
Here I think we want to add a leading underscore (and keep the trailing underscore) to be consistent with the use of header guards elsewhere.